packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -25,16 +25,41 @@ function useSecondaryText() {
 		selectedBlock,
 		getBlockParentsByBlockName,
 		getBlockWithoutInnerBlocks,
+		hoveredBlockIds,
+		getBlockName,
+		getBlock,
 	} = useSelect( ( select ) => {
+		const {
+			getSelectedBlock,
+			getHoveredBlocks,
+			getBlockParentsByBlockName: _getBlockParentsByBlockName,
+			__unstableGetBlockWithoutInnerBlocks,
+			getBlockName: _getBlockName,
+			getBlock: _getBlock,
+		} = select( 'core/block-editor' );
 		return {
-			selectedBlock: select( 'core/block-editor' ).getSelectedBlock(),
-			getBlockParentsByBlockName: select( 'core/block-editor' )
-				.getBlockParentsByBlockName,
-			getBlockWithoutInnerBlocks: select( 'core/block-editor' )
-				.__unstableGetBlockWithoutInnerBlocks,
+			selectedBlock: getSelectedBlock(),
+			getBlockParentsByBlockName: _getBlockParentsByBlockName,
+			getBlockWithoutInnerBlocks: __unstableGetBlockWithoutInnerBlocks,
+			hoveredBlockIds: getHoveredBlocks(),
+			getBlockName: _getBlockName,
+			getBlock: _getBlock,
 		};
 	} );
 
+	// Go through hovered blocks and see if one is of interest.
+	const hoveredTemplatePartBlockId = hoveredBlockIds.find(
+		( blockId ) => getBlockName( blockId ) === 'core/template-part'
+	);
+	if ( hoveredTemplatePartBlockId ) {
+		return {
+			label: getBlockDisplayText(
+				getBlock( hoveredTemplatePartBlockId )
+			),
+			isActive: true,
```

Comment:
Not sure exactly where the designs will take this, but my thought was that `isActive` could be false in this case

Reply:
Oh interesting thought, I totally missed that idea.  So the hovered item will be inactive.  We can do something like this, but I think we would need to complicate the code in this `useSecondayText` hook a bit.  Currently, hover takes priority over selection, so if we have a template part (or child) selected, it will read as non-active if the cursor is over it as well (if we are returning `isActive: false` here).  So we will need to restructure the hook with that in mind, and return `isActive` in the hover state depending on whether or not the hovered item we want to show corresponds to the currently selected one.  Perhaps a good refactoring for a follow up?

Reply:
Sure!

Reply:
>Perhaps a good refactoring for a follow up?

Or not 🤣  ... just refactored to handle this in - 5548826 .  So the hovered item will be inactive if it doesn't correspond to the selected one.